src/pretix/base/services/orders.py

```diff
@@ -515,6 +515,42 @@ def send_expiry_warnings(sender, **kwargs):
             else:
                 o.log_action('pretix.event.order.expire_warning_sent')
 
+@receiver(signal=periodic_task)
+def send_download_reminders(sender, **kwargs):
+    eventcache = {}
+    today = now().replace(hour=0, minute=0, second=0)
+
+    for o in Order.objects.filter(expires__gte=today + timdelta(days=5), download_reminder_sent=False,
```

Comment:
@raphaelm Hi, I am still not sure what should be done here, some help would be appreciated